specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/definitions.json

```diff
@@ -788,7 +794,9 @@
             "type": "string",
             "enum": [
```

Comment:
will be great to have an example here on how to create a websocket based api
`[https://github.com/Azure/azure-rest-api-specs/blob/3f6e9241c599656f29b56d638798f31a6df9338f/specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-01-01-preview/apimapis.json#L226`](https://github.com/Azure/azure-rest-api-specs/blob/3f6e9241c599656f29b56d638798f31a6df9338f/specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-01-01-preview/apimapis.json#L226`)